precompiles/batch/src/mock.rs

```diff
@@ -204,7 +204,7 @@ impl ExtBuilder {
 		let mut ext = sp_io::TestExternalities::new(t);
 		ext.execute_with(|| {
 			System::set_block_number(1);
-			pallet_evm::Pallet::<Runtime>::create_account(
+			let _ = pallet_evm::Pallet::<Runtime>::create_account(
```

Comment:
Since this is in tests probably you should just unwrap here, or `assert_ok` if it works